client/state/data-layer/wpcom/sites/comments/likes/mine/delete/index.js

```diff
@@ -0,0 +1,48 @@
+/**
+ * External dependencies
+ */
+import { translate } from 'i18n-calypso';
+
+/**
+ * Internal dependencies
+ */
+import {
+	COMMENTS_LIKE,
+	COMMENTS_UNLIKE,
+} from 'state/action-types';
+import { http } from 'state/data-layer/wpcom-http/actions';
+import { local } from 'state/data-layer/utils';
+import { dispatchRequest } from 'state/data-layer/wpcom-http/utils';
+import { errorNotice } from 'state/notices/actions';
+
+export const unlikeComment = ( { dispatch }, action ) => {
+	dispatch( http( {
+		method: 'POST',
+		apiVersion: '1.1',
+		path: `/sites/${ action.siteId }/comments/${ action.commentId }/likes/mine/delete`
+	}, action ) );
+};
+
+export const updateCommentLikes = ( { dispatch }, { siteId, postId, commentId }, next, { like_count } ) => dispatch( local( {
+	type: COMMENTS_UNLIKE,
```

Comment:
We can leave this for another PR, but I feel like this would be clearer with an explicit action. For example `COMMENT_COUNT_SET`, if we periodically want to sync the count from network responses.